### PR TITLE
Feature/blueprint for auth bookmarks 001

### PR DESCRIPTION
**Pull request by oswinrozario**
[Implemented basic blueprint in bookmarks.py. Also made changes in app.py file to register those blueprint. the registered routes are tested with sample dummp response]

[Created controller directory. added auth.py and bookmarks.py. implemented basic auth blueprint in auth.py]

### CODE REVIEW

File: src/controllers/auth.py
Comment (sg092000) [inline src/controllers/auth.py:1]: include try-except block

File: src/controllers/bookmarks.py
Comment (sg092000) [inline src/controllers/bookmarks.py:1]: include try-except block

File: src/controllers/auth.py
Comment (oswinrozario) [reply src/controllers/auth.py:1]: included basic try catch functionality successfully.

File: src/controllers/bookmarks.py
Comment (oswinrozario) [reply src/controllers/bookmarks.py:1]: included basic try catch functionality successfully.

**Pull request merged**